### PR TITLE
align with english version

### PR DESCRIPTION
**Pull request by Fale**
Align the i18n strings with English ones, more specifically:

1. Sort them in alphabetical order (as they are in the English one)
2. Translate the missing strings

### CODE REVIEW

File: i18n/it.toml [hunk 1]
```diff
@@ -1,82 +1,185 @@
...
+other = "Questo glossario vuole essere una lista comprensiva per standardizzare la terminologia di Kubernetes. Include termini tecnici che sono specifici di Kubernetes, così come termini più generali che sono utili per dare un contesto."
+
+[layouts_docs_glossary_deselect_all]
+other = "Deseleziona tutto"
+
+[layouts_docs_glossary_click_details_after]
+other = "per avere una spiegazione più completa per il termine."
+
+[layouts_docs_glossary_click_details_before]
+other = "Fare click sull'icona "
```
[the remark this answers is not in the document]
Comment (sftim) [reply i18n/it.toml]: The [original](https://github.com/kubernetes/website/blob/65d36413fb23dffd2c0f3bbdb73b4cc5f71d2745/i18n/en.toml#L97-L98) has whitespace there too.

File: i18n/it.toml [hunk 2]
```diff
@@ -1,82 +1,185 @@
...
+other = "Filtra i termini sulla base delle loro etichette"
+
+[layouts_docs_glossary_select_all]
+other = "Seleziona tutto"
+
+[layouts_docs_partials_feedback_improvement]
+other = "suggerire un miglioramento"
+
+[layouts_docs_partials_feedback_issue]
+other = "Apri un issue sul repository GitHub se vuoi "
```
[the remark this answers is not in the document]
Comment (Fale) [reply i18n/it.toml]: the English one has a trailing whitespace

**Comment by sftim** [comment]
@kubernetes/sig-docs-it-owners please add an /approve to mark that you're happy with this

**Comment by Fale** [comment]
@sftim Fabrizio which is in that group, I assume (https://github.com/kubernetes/website/blob/master/OWNERS_ALIASES#L128) already added `/approve`. I suppose something else is still needed

**Comment by rlenferink** [comment]
@sftim since this touches a file outside of the `content/it` it also needs sign-off from an English owner (e.g. yourself @sftim). https://github.com/kubernetes/website/pull/19687#pullrequestreview-380979171 already contained `/approve` from an it-owner.

**Comment by sftim** [comment]
Ah, sorry I missed https://github.com/kubernetes/website/pull/19687#pullrequestreview-380979171
/approve

**Pull request merged**